examples/shell/cmd_device.cpp

```diff
@@ -494,6 +494,84 @@ int cmd_device_get(int argc, char ** argv)
     return error;
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_WPA
+int cmd_device_sta(int argc, char ** argv)
+{
+    streamer_t * sout = streamer_get();
+
+    CHIP_ERROR error = CHIP_NO_ERROR;
+
+    VerifyOrExit(argc > 0, error = CHIP_ERROR_INVALID_ARGUMENT);
+
+    VerifyOrExit(PlatformMgr().TryLockChipStack(), error = CHIP_ERROR_INVALID_ARGUMENT);
+
+    if (strcmp(argv[0], "mode") == 0)
+    {
+        const char * typeStr                      = u8"Unknown";
```

Comment:
u8 does nothing here since this is an ASCII string. Should we really be adding this for ASCII strings?

Reply:
Acked.